From f39e7bfc5ed07b5ecaeb705c4eae4855ca120d47 Mon Sep 17 00:00:00 2001
From: Oleksiy Obitotskyy <oobitots@cisco.com>
Date: Wed, 25 Mar 2020 21:21:35 +0200
Subject: [PATCH 05/12] qemu: Do not include file if not exists

Script configure checks for if_alg.h and check failed but
if_alg.h still included.

Upstream-Status: Submitted [https://lists.gnu.org/archive/html/qemu-devel/2020-03/msg07188.html]
Signed-off-by: Oleksiy Obitotskyy <oobitots@cisco.com>

[update patch context]
Signed-off-by: Sakib Sajal <sakib.sajal@windriver.com>

---
 linux-user/syscall.c | 2 ++
 1 file changed, 2 insertions(+)

Index: qemu-8.0.0/linux-user/syscall.c
===================================================================
--- qemu-8.0.0.orig/linux-user/syscall.c
+++ qemu-8.0.0/linux-user/syscall.c
@@ -115,7 +115,9 @@
 #include <linux/blkpg.h>
 #include <netpacket/packet.h>
 #include <linux/netlink.h>
+#if defined(CONFIG_AF_ALG)
 #include <linux/if_alg.h>
+#endif
 #include <linux/rtc.h>
 #include <sound/asound.h>
 #ifdef HAVE_BTRFS_H
